python/tests/test_account.py

```diff
@@ -1821,6 +1821,7 @@ def test_undecipherable_group(self, acfactory, lp):
         # Error message should be assigned to the chat with ac1.
         lp.sec("ac4: checking that message is assigned to the sender chat")
         error_msg = ac4._evtracker.wait_next_incoming_message()
+        assert error_msg.error  # There is an error decrypting the message
```

Comment:
@link2xt you could check for the correct error message as it is hardcoded/not translated

Reply:
This discouraged in the docs: https://github.com/deltachat/deltachat-core-rust/blob/09113e2579c57f76c91e3f51376c086ad55f0871/deltachat-ffi/deltachat.h#L3554 (of course this doesn't mean we can't do it as it was just some other dev who wrote it :) but I still wouldn't do it as this would make changing the error msg harder)

Reply:
you are right, it should not be parsed in the python test case. I think it's very unfortunate that it's a freeform text in the first place for some errors which are fixed. Sometimes it can contain free form error messages, but errors like the encryption error are fixed, maybe these exceptions should be mentioned in the documentation :)